src/app/Dashboard.js

```diff
@@ -69,6 +69,12 @@ class Dashboard extends Component<Props> {
         to: tokenUrl + '/sto',
         isActive: location.slice(-4) === '/sto',
       },
+      {
+        title: 'Whitelist',
+        icon: <img src={icoHandshake} alt='Whitelist' />,
+        to: tokenUrl + '/whitelist',
+        isActive: location.slice(-10) === '/sto',
```

Comment:
wrong condition, should be /whitelist instead of /sto

Reply:
fixed